test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -200,3 +200,7 @@ estimate: 90m
 
     In an earlier step the presence of the RHOAMApiUsageOverLimit was verified and a time noted. If 30 minutes has passed
     since then please verify that the alert is firing and that an email has been received to the BU and Customer email.
+
+18. Revert the value changed as part of this test case
+
+Use the commands from Step #4 and Step #5, just use the original values. Only required if the cluster is to be used for other test cases.
```

Comment:
Not sure if reverting the changes is needed, because this test case is performed on a separate cluster (see the step n. 2)

Would it make sense to change the environment to `external` for this test case instead?

Reply:
Agree, better to have this external. Will update.